src/base/bittorrent/session.cpp

```diff
@@ -3051,6 +3055,22 @@ void Session::setAsyncIOThreads(const int num)
     configureDeferred();
 }
 
+int Session::checkingMemUsage() const
+{
+    return qBound(1, m_checkingMemUsage.value(), 1024); // 1024 MiB = 1 GiB
+}
+
+void Session::setCheckingMemUsage(int size)
+{
+    size = qMin(size, 1024);  // 1024 MiB = 1 GiB
```

Comment:
I remember asking about that earlier. Why is there only an upper limit?

Reply:
It seems we don't really need an upper limit.

Reply:
I updated this part and forced pushed. (for the last time)